deploy/examples/cluster.yaml

```
@@ -106,6 +106,9 @@ spec:
     #ipFamily: "IPv6"
     # Ceph daemons to listen on both IPv4 and Ipv6 networks
     #dualStack: false
+    #multiClusterService:
+    # enable: false
```

While we're not completely consistent in rook about this term, I vote for `enabled`.
```suggestion
    # enabled: false
```